src/js/sequences/components/Field.jsx

```diff
@@ -1,29 +1,27 @@
 import PropTypes from "prop-types";
 import React from "react";
 import styled from "styled-components";
-import { Badge, InputContainer, InputError, InputGroup, InputLabel, TextArea } from "../../base";
+import { Badge, InputError, InputGroup, InputLabel, TextArea } from "../../base";
 export const SequenceFieldTextArea = styled(TextArea)`
     font-family: ${props => props.theme.fontFamily.monospace};
-    transform-text: uppercase;
+    text-transform: uppercase;
```

Comment:
This only gives the appearance of uppercase without actually upper casing the letters in state. So the server still gets lower case letters which it won't accept